_includes/header.html

```diff
@@ -51,22 +54,16 @@
                         </a>
                         <ul class="dropdown-menu">
                             <li>
-                                <a href="/projects" title="Discover projects that use ev3dev">Projects</a>
-                            </li>
-                            <li>
-                                <a href="/share" title="Share projects that use ev3dev">Share</a>
+                                <a href="/projects" title="Discover projects that use ev3dev">Project showcase</a>
```

Comment:
I think this depends on what the "projects" page becomes. If we expand it to be a full-fledged "examples" hub, the link to that (which may be the same page or different) should definitely be obvious. As it is now though, I'll flip it around.